tensorflow_serving/model_servers/server_core.h

```diff
@@ -117,6 +117,9 @@ class ServerCore : public Manager {
     // Time interval between file-system polls, in seconds.
     int32 file_system_poll_wait_seconds = 30;
 
+    // Timeout for waiting on a servale to reach state.
```

Comment:
Don't you want to add a flag to model_servers/main.cc so you can control this from the command line when running the model-server binary?